src/tests/krate.rs

```diff
@@ -1284,61 +1270,50 @@ fn diesel_not_found_results_in_404() {
 
 #[test]
 fn following() {
-    #[derive(Deserialize)]
-    struct F {
-        following: bool,
-    }
-
-    let (_b, app, middle) = app();
-    let mut req = req(Method::Get, "/api/v1/crates/foo_following/following");
-
-    let user;
-    {
-        let conn = app.diesel_database.get().unwrap();
-        user = new_user("foo").create_or_update(&conn).unwrap();
-        sign_in_as(&mut req, &user);
-        CrateBuilder::new("foo_following", user.id).expect_build(&conn);
-    }
+    // TODO: Test anon requests as well?
```

Comment:
Yeah we probably should. And once there are more tests that exercise following/unfollowing (i'm surprised but not really surprised this is the only one), it'd be nice to extract the closures into functions that take the user and crate name. But I think that can wait.